zarr/tests/test_storage.py

```diff
@@ -1341,6 +1341,22 @@ def test_s3_complex(self):
         )
         assert (a[:] == -np.ones((8, 8, 8))).all()
 
+    def test_exceptions(self):
+        import fsspec
+        m = fsspec.filesystem("memory")
+        g = zarr.open_group("memory://test/out.zarr", mode='w')
+        arr = g.create_dataset("data", data=[1, 2, 3, 4],
+                               dtype="i4", compression=None, chunks=[2])
```

Comment:
```suggestion
        arr = g.create_dataset("data", data=[1, 2, 3, 4], dtype="i4", compression=None, chunks=[2])
```
black/ruff line length is 100, this will fix one red x